degiro updates

Joren-vanGoethem opened the pull request:
added 'sell' detection
and make GetQuantity and GetUnitPrice currency agnostic and buy/sell agnostic

VibeNL commented:
Thanks for the contribution. I will have a closer look tomorrow, but could you at least add a testcase for the sell activity?

Joren-vanGoethem commented:
will do, btw add some labels for PR's like feature/bug/enhancement/... ;)

Joren-vanGoethem commented:
i added a few unit tests, i think the 'example' naming you used for directories of csv files is quite useless to be honest
'example' doesn't tell you anything about the files in that directory
i added very small and simple unit tests that only test one simple thing like buy euro older, sell euro order, ...

VibeNL commented on GhostfolioSidekick/FileImporter/DeGiro/DeGiroParser.cs:
```diff
@@ -146,17 +150,24 @@ protected override CsvConfiguration GetConfig()
 		}
 
-		private decimal GetUnitPrice(DeGiroRecord record)
+		private decimal GetUnitPrice(DeGiroRecord record, bool buy = true)
```
not sure why there is an unused parameter 👼

Joren-vanGoethem replied:
you'll have to approve the new changes

VibeNL reviewed (approved):
Just a single unused parameter, nothing major :) . Looks good!

VibeNL commented:
> i added a few unit tests, i think the 'example' naming you used for directories of csv files is quite useless to be honest 'example' doesn't tell you anything about the files in that directory i added very small and simple unit tests that only test one simple thing like buy euro order, sell euro order, ...

IT is a work in progress, I like the suggestion 👍 . BTW, the build fails, likely on a culture problem

VibeNL commented:
I will merge it and fix it in master. Need this code for another change :)
